modules/logging-content-filter-drop-records.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-content-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-content-filter-drop-records_{context}"]
+= Configuring content filters to drop unwanted log records
+
+When content filters are configured, the log collector edits log streams according to the filters before forwarding. If you configure the `drop` filter, the collector drops unwanted log records that match the specified configuration.
```

Comment:
maybe also counter to prune... explicitly say... "When drop content filters are configured" or "drop filters"